### DISCUSSION TITLE
[Success Story] Connecting to remote Macs using Apple Internet Router #GlobalTalk

### DISCUSSION THREAD

**Discussion by marciot** [akuker/RASCSI#1441] [Show and tell]
# Info

- Which version of Pi are you using: Raspberry Pi W
- Which github revision of software:  v23.11.01
- Which board version: RaSCSI 2.4a
- Which computer is the PiSCSI connected to: Macintosh LC II 
- Which OS you are using (output of 'lsb_release -a'): Raspbian GNU/Linux 11 (bullseye)

# Describe the issue

Not an issue! Just sharing my success in participating in #GlobalTalk using PiSCSI!

# What is it?

GlobalTalk is an effort to interconnect vintage Macs across the Internet using AppleTalk and Apple Internet Router software. You can read about it here:

https://marchintosh.com/globaltalk.html

Or, watch this video:

https://www.youtube.com/watch?v=cIMmz0EWWi0

# Steps to get it to work on PiSCSI:

1) Request access to GlobalTalk spreadsheet using the following page:

https://marchintosh.com/globaltalk.html

2) Enable transparent bridging on PiSCSI using the instructions from the following thread:

https://github.com/PiSCSI/piscsi/issues/1387

This is necessary for the transparent forwarding of UDP packets in the VPN tunnel

3) Install the "HD1-Apple Internet Router v2.hda.zip" image (there is a link to the download in this document):

https://docs.google.com/document/d/1pXMjrAF5vC08TamkdSt2oFCrDFMP47i5dYcjshn9JzU/edit

4) MacTCP is not compatible with modern DHCP servers, so configure it use "Manual" addressing and set it to an unused IP address. Also put in your router address and DNS.

5) Configure your firewall to forward UDP 387 packets to the address you configured in MacTCP

6) Configure Apple Internet Router as shown in the following document:

https://biosrhythm.com/?p=2767

I did things slightly differently, setting up two zones, one for LocalTalk and another for EtherTalk, this may not be necessary, but since this is what worked for me, I went with it:

![AIR Config](https://github.com/PiSCSI/piscsi/assets/6752228/2b924822-8e54-4dd0-bfe8-9eeceda98109)

**Comment by rdmark** [comment]
Thank you for sharing your setup! GlobalTalk is a such neat idea, and it's encouraging to know that piscsi can serve as a piece of the puzzle.

Converting your ticket from Issue to Discussion. I hope you agree with the change.

**Reply by marciot** [reply]
@rdmark No worries, but it's a (minor) issue it doesn't work right out of the box! Let's get that transparent bridging in the configurator script somehow, it's pretty awesome! 😄
